src/endpoints/seed/index.ts

```diff
@@ -1136,23 +1143,3 @@ export const innerSeed = async ({
 
   payload.logger.info('Seeded database successfully!')
 }
-
-async function fetchFileByURL(url: string): Promise<File> {
```

Comment:
This is used by other non biography photos starting on like `726` so maybe we should move this under a utility file instead of nesting it under `./biographies`